api/src/common/constant.ts

```diff
@@ -0,0 +1,14 @@
+export const LanguageMap: Map<string, string> = new Map<string, string>([
+  ['ASL', 'Asl'],
+  ['CART', 'Cart'],
+]);
+
+const reverseMap = (map: Map<string, string>) => {
+  const temp = new Map<string, string>();
+  for (const [key, value] of map) {
+    temp.set(value, key);
+  }
+  return temp;
+};
+
+export const ReverseLanguageMap: Map<string, string> = reverseMap(LanguageMap);
```

Comment:
I think a descriptively named function would be a better export, like `fixLanguageName`. `ReverseLanguageMap.get` isn't very clear what it's doing.

Reply:
Remove this variable fix this in front end but also rename the other one.